components/automate-chef-io/content/docs/saml.md

```diff
@@ -42,12 +42,44 @@ Users who sign in via SAML will have a session time of 24 hours before needing t
 
 ## Supported Identity Management Systems
 
+- [Azure AD]({{< relref "#azure-ad" >}})
 - Office365
 - OKTA
 - OneLogin
 - Ping
 - Tivoli Federated Identity Manager
 
+In the following section, you can find further details about specific supported identity management systems.
+
+### Azure AD
+
+For using Chef Automate's SAML integration with Azure AD, please note that **signing key rotation** is
+not supported to be done automatically. The signing certificate used with Azure for SAML sign-ins
+has to be **managed manually**.
+
+After setting up your application as a _"non-gallery application"_, configure its SAML sign-on method.
+For both _Identifier (Entity ID)_ and _Reply URL (Assertion Consumer Service URL)_, enter
+`https://{{< example_fqdn "automate" >}}/dex/callback`.
```

Comment:
Do we want to guide them to _where_ they should be "setting up your application" or is this obvious to someone using Azure AD?

Reply:
I'm not sure. What do you think? 👇 

![Screenshot 2020-07-27 at 15 30 39](https://user-images.githubusercontent.com/870638/88548286-33cf2b00-d01f-11ea-806b-35887a622e52.png)

Here, we select "View" below the middle tile, and get to

![Screenshot 2020-07-27 at 15 32 09](https://user-images.githubusercontent.com/870638/88548381-506b6300-d01f-11ea-95c8-c731cfe03cb5.png)

...where clicking on "Enterprise application" will get us to

![Screenshot 2020-07-27 at 15 32 15](https://user-images.githubusercontent.com/870638/88548471-6ed15e80-d01f-11ea-8648-868f5eae460f.png)

where we find the "non-gallery application".

I'm sure there's other ways. I think I'd like to avoid adding too much of that here, since then our docs are tied to a UI in Azure; which may change at some point.

Reply:
Would this piece of Azure documentation be relevant and a possible link to include? https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal#add-an-app-to-your-azure-ad-tenant

Reply:
Hmm I don't know about that specific link, it seems to be a little misleading: it's describing the github integration, which is easier to set up.

Do you prefer having some official docs to link to? I can try digging up a more fitting link.

Reply:
I know we won't find specific docs referring to Chef, but it would be good to link official docs that covers the general aspects of Azure's "adding an application".  That way, our Chef Automate documentation can provide the needed details rather than having to recreate the wheel with detailed documentation that may change.

Reply:
Yeah we're on the same page here. I'll go look for a generic docs page that we can reference. :)